### DISCUSSION TITLE
Try the new `bomb` command!

### DISCUSSION THREAD

**Discussion by winstxnhdw** [winstxnhdw/lc-hax#359] [Announcements]
I've always wanted to do this. I spent 8 hours exhausting every code combination for grabbing and finally got it to work. It's hilarious.

**Comment by Dexteriority** [comment]
I didn't know this update, and i type it while someone using jetpack, i thought it was coincidence that they exploded but no. its the script 😂😂😂😂😂😂😂

**Comment by joep26020** [comment]
> I've always wanted to do this. I spent 8 hours exhausting every code combination for grabbing and finally got it to work. It's hilarious.

ive been doing this with the lockdown lobbies its amazing!

**Reply by winstxnhdw** [reply]
Try out `bombard` and `destroy —all` as well. They work really well in lockdown lobbies. Best thing—it’s going to be tough if they want to lock it down. First in the world stuff.

**Reply by joep26020** [reply]
lol will do

**Reply by joep26020** [reply]
what does bombard do?

**Reply by winstxnhdw** [reply]
Sends every jetpack on the map and carpets the area around the player.

**Reply by joep26020** [reply]
like makes it blow up or fly? and by carpets do you mean the item

**Reply by winstxnhdw** [reply]
https://en.m.wikipedia.org/wiki/Carpet_bombing

**Reply by joep26020** [reply]
LMAO
